index.js

```diff
@@ -1 +1,133 @@
-module.exports = require('./lib/emoji');
\ No newline at end of file
+'use strict'
+
+const charRegex = require('char-regex')()
+const emojiRegex = require('emoji-regex')()
+const randomItem = require('random-item')
+const skinTone = require('skin-tone')
+const is = require('@sindresorhus/is')
+const { assert } = is
+
+const emojiData = Object.entries(require('emojilib').lib).map(([name, { char: emoji }]) => [name, emoji])
```

Comment:
I personally find it confusing to have `require`s nested within other statements. It'll also be harder to convert this to ESM if & when the time comes.